assignment-1/cause_3.md

```diff
@@ -0,0 +1,58 @@
+# Change Management
+
+In order to shop for a healthcare plan, consumers had to register on the site for an account first. 
+It was reported that the response times of the registration pages were very high with load times up to 71 seconds. [2]
+Analysis showed that the architecture was not able to handle the concurrent users for the registration pages. 
+Around September 2013, three months before the deadline, the decision was made that all users had to register first, before they could start shopping for a health plan. [3] Because of this change the registration pages became a bottleneck for the entire site. The changing functional requirement had a major impact on the architecture of the site.

```

Comment:
The site went live in October, which deadline are you actually talking about here?